src/renderer/ducks/register/slice.ts

```diff
@@ -67,6 +68,9 @@ export const registerSlice = createSlice({
         setPassword: (state, action: PayloadAction<string>): void => {
             state.password = action.payload;
         },
+        setName: (state, acton: PayloadAction<string>): void => {
+            state.name = acton.payload;
```

Comment:
I think you should update keystore as well?

Reply:
here is stored temporarily so after can be stored at Keystore